mirgecom/initializers.py

```diff
@@ -292,7 +292,7 @@ class Lump:
     """
 
     def __init__(
-            self, *, dim=1, nspecies=0,
+            self, *, dim=1,
```

Comment:
I don't understand this part of the diff.  When I look at your branch, the `nspecies` is there, but this diff seems to suggest it was removed?

Reply:
Sorry, this part is unrelated to the PR title. I saw that `Lump` had an `nspecies` parameter that wasn't used or documented, so I got rid of it (https://github.com/illinois-ceesd/mirgecom/pull/460/commits/b50fc0e6faf1de3ba120c94d752353b60dad28e9).

Reply:
Laugh, I just didn't read the diff very well.  The fact that this was a whole different initialization just escaped my attention altogether.